contrib/pg_tde/expected/partition_table.out

```diff
@@ -0,0 +1,92 @@
+CREATE EXTENSION pg_tde;
+SELECT pg_tde_add_global_key_provider_file('global_keyring_provider','/tmp/pg_tde_keyring.per');
```

Comment:
Do not use the global  provider unless there is a specific point to it. Otherwise you risk order dependent tests unnecessarily like what happened here..

Reply:
Fixed